INSTALL.md

```diff
@@ -206,15 +206,15 @@ by the official GMT repository. You can uninstall the older packages by:
 
 ### Ubuntu/Debian
 
-**Note:** Ubuntu/Debian are way behind in packing a recent GMT version.
-Typically you may find they offer 5.2.1 from 2015 while the rest of us have
-moved on to 2019. Your best bet then is to
-[build the latest release from source](BUILDING.md).
-Otherwise, installing from the distros goes like this:
+GMT 6.0 packages are available for Ubuntu 20.04 (Focal Fossa) and Debian 11 (Bullseye/Testing).
+Install it via
 
-Install GMT5 via
+    sudo apt-get install gmt gmt-dcw gmt-gshhg-data
```

Comment:
I think it's `gmt-gshhg`, not `gmt-gshhg-data`.

Reply:
Oops, you're right. Was looking at the virtual package somehow.